cranelift/module/src/module.rs

```diff
@@ -520,24 +552,18 @@ pub trait Module {
     fn declare_data_in_func(&self, data: DataId, func: &mut ir::Function) -> ir::GlobalValue {
         let decl = &self.declarations().data_objects[data];
         let colocated = decl.linkage.is_final();
+        let user_name_ref = func.declare_imported_user_function(ir::UserExternalName {
+            namespace: 1,
+            index: data.as_u32(),
+        });
         func.create_global_value(ir::GlobalValueData::Symbol {
-            name: ir::ExternalName::user(1, data.as_u32()),
+            name: ir::ExternalName::user(user_name_ref),
             offset: ir::immediates::Imm64::new(0),
             colocated,
             tls: decl.tls,
         })
     }
 
-    /// TODO: Same as above.
-    fn declare_func_in_data(&self, func: FuncId, ctx: &mut DataContext) -> ir::FuncRef {
-        ctx.import_function(ir::ExternalName::user(0, func.as_u32()))
-    }
-
-    /// TODO: Same as above.
-    fn declare_data_in_data(&self, data: DataId, ctx: &mut DataContext) -> ir::GlobalValue {
-        ctx.import_global_value(ir::ExternalName::user(1, data.as_u32()))
-    }
```

Comment:
What are calls to these supposed to be replaced with?

Reply:
I managed to put them back eventually.